### PR TITLE
Fwm qs

### PR DESCRIPTION
**Pull request by vhorne**
# Best Practice Checklist

Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment.
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: fwm-docs-qs/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+               "description": "Password for the admin account on the servers"
+           }
+       }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/virtualWans",
+            "apiVersion": "2019-08-01",
+            "name": "VWan-01",
+            "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline fwm-docs-qs/azuredeploy.json:23]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: fwm-docs-qs/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,464 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/deploymentTemplate.json#",
```
Comment (bmoore-msft) [inline fwm-docs-qs/azuredeploy.json:2]: use a valid schema (see test results or VS Code)

File: fwm-docs-qs/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+            "type": "Microsoft.Compute/virtualMachines",
+            "apiVersion": "2019-07-01",
+            "name": "Jump-Srv",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/networkInterfaces', 'netInterface-jump-srv')]"
+            ],
+            "properties": {
+                "hardwareProfile": {
+                    "vmSize": "Standard_D2s_v3"
```
Comment (bmoore-msft) [inline fwm-docs-qs/azuredeploy.json:193]: use a parameter for the vm sizes

File: fwm-docs-qs/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+                   }
+               ]
+           }
+       },
+       {
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "apiVersion": "2020-03-01",
+            "name": "nsg-workload-srv",
+            "location": "[resourceGroup().location]",
+            "properties": {
```
Comment (bmoore-msft) [inline fwm-docs-qs/azuredeploy.json:371]: add security rules

File: fwm-docs-qs/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,464 @@
...
+                            "nextHops": [
+                                "10.1.64.4"
+                            ]
+                    }
+                    ],
+                        "attachedConnections": [
+                           "All_Branches"
+                        ]
+                },
+                "dependsOn": [
```
Comment (bmoore-msft) [inline fwm-docs-qs/azuredeploy.json:456]: move all dependsOn properties above the properties property

File: fwm-docs-qs/azuredeploy.json (hunk 1 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:23]: done

File: fwm-docs-qs/azuredeploy.json (hunk 2 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:2]: done

File: fwm-docs-qs/azuredeploy.json (hunk 4 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:371]: There are no specific security rules for this VM. It has no public IP address and no specific open ports.

File: fwm-docs-qs/azuredeploy.json (hunk 3 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:193]: done

File: fwm-docs-qs/azuredeploy.json (hunk 1 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:23]: done

File: fwm-docs-qs/azuredeploy.json (hunk 5 above)
Comment (vhorne) [reply fwm-docs-qs/azuredeploy.json:456]: done

File: fwm-docs-qs/azuredeploy.json (hunk 4 above)
Comment (bmoore-msft) [reply fwm-docs-qs/azuredeploy.json:385]: why do we have an NSG then?

**Pull request merged**